sklearn/compose/_column_transformer.py

```diff
@@ -172,6 +172,12 @@ class ColumnTransformer(TransformerMixin, _BaseComposition):
 
         .. versionadded:: 0.24
 
+    feature_names_in_ : ndarray of shape (`n_features_in_`,)
+        Names of features seen during :term:`fit`. Only defined if the
+        underlying estimators expose such an attribute when fit.
```

Comment:
I think you mean to say "Only defined if X has feature names during `fit`"? This doesn't have anything to do with the underlying estimators 😁

Reply:
wrong copy-paste